reference-implementation/web-platform-tests/readable-streams/general.js

```diff
@@ -608,23 +608,35 @@ promise_test(() => {
 
 }, 'ReadableStream pull should be able to close a stream.');
 
-test(() => {
+promise_test(t => {
```

Comment:
- IIRC, the following two test are just for increasing test coverage. these pass even on the current master branch
- enqueue() has been changed not to throw the stored error. So, I've removed the last test. This is related to #424.